src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -337,8 +337,11 @@ bool PolicyHandler::LoadPolicyLibrary() {
 bool PolicyHandler::CreateManager() {
   void* policy_handle = dlopen(kLibrary.c_str(), RTLD_LAZY);
   const char* error = dlerror();
-  if (error) {
-    LOG4CXX_ERROR(logger_, error);
+  if (!policy_handle) {
```

Comment:
`const char* error = dlerror();` isn't needed outside this block

Reply:
Technically speaking you are correct, though it seems standard to call `dlerror` after a related operation. This deque's the first unread error message, if available. It might not be relevant in this case, but I don't see much harm in including it here